.gitignore

```diff
@@ -97,4 +97,3 @@ Cargo.toml
 
 # Temporary paket files for C#
 paket-files/
-paket.lock
```

Comment:
https://fsprojects.github.io/Paket/faq.html#Why-should-I-commit-the-lock-file

Reply:
Yeah we do normally commit the lock files for various languages' package dependencies!